zkvm/spec/ZkVM.md

```diff
@@ -182,10 +182,10 @@ Data cannot be larger than the entire transaction program and cannot be longer t
 
 ### Contract type
 
-A contract is a [predicate](#predicate) and a [payload](#contract-payload) guarded by that predicate.
+A contract is a [predicate](#predicate) and a [payload](#contract-payload) . The payload is guarded by the predicate.
 
 Contracts are created with the [`contract`](#contract) instruction and
-destroyed by evaluating the predicate, leaving their stored items on the stack.
+destroyed by evaluating the predicate, leaving their payload on the stack.
```

Comment:
I changed this to payload, because if the user did not click the definition for payload, they may not know that they are "stored items".